mlos_bench/mlos_bench/tests/service/remote/azure/azure_fileshare_test.py

```diff
@@ -16,21 +18,21 @@
 
 @patch("mlos_bench.service.remote.azure.azure_fileshare.open")
 @patch("mlos_bench.service.remote.azure.azure_fileshare.os.makedirs")
-def test_download_file(mock_makedirs, mock_open, azure_fileshare):
+def test_download_file(mock_makedirs: MagicMock, mock_open: MagicMock, azure_fileshare: AzureFileShareService) -> None:
     filename = "test.csv"
     remote_folder = "a/remote/folder"
     local_folder = "some/local/folder"
     remote_path = f"{remote_folder}/{filename}"
     local_path = f"{local_folder}/{filename}"
     # pylint: disable=protected-access
     mock_share_client = azure_fileshare._share_client
-    mock_share_client.get_directory_client.return_value = Mock(
+    mock_share_client.get_directory_client.return_value = Mock(     # type: ignore[attr-defined]
```

Comment:
btw is there a way to define that `# type: ignore[attr-defined]` once for the entire file? I'd be totally fine with it in this case

Reply:
https://github.com/python/mypy/issues/6713